tests/test_java_wrapper.py

```diff
@@ -49,3 +50,11 @@ def test_sign_nanopub_no_rsa_key(tmp_path):
     with pytest.raises(RuntimeError) as e:
         java_wrapper.sign(unsigned_file=temp_unsigned_file)
     assert 'RSA key appears to be missing' in str(e.value)
+
+
+@pytest.mark.flaky(max_runs=10)
+@skip_if_nanopub_server_unavailable
```

Comment:
This will actually check if garlic server is available, I would propose to just leave this for now to reduce complexity.

Reply:
Agreed.